### ISSUE TITLE
Use link reference to css file instead of inline style option?

### ISSUE THREAD

**Issue by Norlandz** [yzhang-gh/vscode-markdown#1429]
### Proposal

Use link reference to css file instead of inline style option?

I rather to have `<link rel="stylesheet" type="text/css" href="file:///c%3A/aaa/markdown.css">` instead of inline `<style>`. 
Is there an option to change it?

- I dont want to constantly press save to regenerate my html file whenever there is a change in my css file.

### Other information

Kinda like the opposite of https://github.com/yzhang-gh/vscode-markdown/issues/57

**Comment by yzhang-gh** [comment]
You can make use of the `markdown.styles` option (which is provided by VS Code itself). It is used in both VS Code preview and the exported HTML.

**Comment by Norlandz** [comment]
> You can make use of the `markdown.styles` option (which is provided by VS Code itself). It is used in both VS Code preview and the exported HTML.

Yes I know that option, but isnt that is irrelevant? It dose **add** styles to the html. 
But what I want is to have an option for the added style to be **embedded / inline / linked**.

---

My current workaround is just use old style html -- manually adding **`<link css>`** to my md file. 

This seems ok, it also solves my another problem: https://github.com/yzhang-gh/vscode-markdown/issues/1427 Auto print to html when css file is changed.

But still, idk if there is a better design.
